docs/modules/reference/pages/glossary.adoc

```diff
@@ -66,10 +86,17 @@ Kafka:: Clusterable, open-source message broker designed for high-throughput/low
 Kibana:: Kibana is a component of the Elastic Stack (comprised of Elasticsearch, Logstash, and Kibana).
 It provides visualization capabilities for data indexed in an Elasticsearch cluster.
 
-KSC Reports:: Key SNMP customized (KSC) reports provide a way to generate prefabricated graphical views of collected data. 
+KSC Reports:: Key SNMP customized (KSC) reports provide a way to generate prefabricated graphical views of collected data.
 They let you display data from different devices and sources (SNMP, ICMP, HTTP) on one page.
 (See xref:operation:admin/webui/opsboard/dashlet/ksc.adoc#ksc[KSC Reports].)
 
+Kubernetes:: Kubernetes is an open-source container orchestration system for automating software deployments, scaling, and management.
+Originally designed by Google, it is now maintained by the Cloud Native Computing Foundation.
+(See https://kubernetes.io/[Kubernetes].)
+
+Location:: The geographical location of a node that is being managed using OpenNMS.
+Nodes can be manually assigned a location (See xref:reference:provisioning/handlers/dns.adoc[Location]), or the location can be determined using the GeoIP Provisioning Adapter (See xref:reference:provisioning/adapters/geoip.adoc[GeoIP]).
```

Comment:
```suggestion
This is distinct from geolocation, which is the actual physical location of the node. 
```
Feel free to reword my suggestion. The deleted text is great and you could use it in the geolocation definition.